.github/ISSUE_TEMPLATE/1-leak.md

```diff
@@ -13,5 +13,5 @@ This issue tracker is not for help with memory leaks detected by LeakCanary in y
 
 To fix a leak: 
 
-* First, learn the fundamentals: https://github.com/square/leakcanary#fundamentals
-* Then, create a Stack Overflow question: http://stackoverflow.com/questions/tagged/leakcanary
+* First, learn the [fundamentals](https://square.github.io/leakcanary/fundamentals/)
```

Comment:
The issue template isn't a markdown file, the content isn't actually rendered but instead is displayed as default content for the issue description. That's why we're not using markdown for links, so here it should be: `learn the fundamentals: https://square.github.io/leakcanary/fundamentals`

Reply:
I think issues make use of markdown and render it properly, changed it anyway. Let me now if there's anything left.